### ISSUE TITLE
html2image not working Windows 11 with Chrome 128.0.6613.120 (Official Build) (64-bit)

### ISSUE THREAD

**Issue by git-n-pissed** [vgalin/html2image#159]
Using latest version of html2image (2.0.4.3).

This does not work:

```Python
import html2image
hti = html2image.Html2Image(browser='chrome', output_path=r'C:\Users\me\Desktop')
hti.screenshot(url='https://www.python.org', save_as='python_org.png')
```

This does work:

```Python
import html2image
hti = html2image.Html2Image(browser='edge', output_path=r'C:\Users\me\Desktop')
hti.screenshot(url='https://www.python.org', save_as='python_org.png')
```

I don't know when Chrome automatically updated or what version I had before it updated, but my `html2image` code was working less than 2 weeks ago with Chrome.

Update: Edge just updated to version 128.0.2739.67 (Official build) (64-bit), now it isn't working either.  I think issue #158 created by @jrb-github is the same problem I'm seeing, and he has already done some research narrowing down the problem.  Please feel free to close this issue.

**Comment by vgalin** [comment]
A new version of html2image was released (v2.0.5), it should address this issue. Could you confirm that everything is working properly after updating the package? Thanks!
```
pip install --upgrade html2image
```

**Comment by git-n-pissed** [comment]
> A new version of html2image was released (v2.0.5), it should address this issue. Could you confirm that everything is working properly after updating the package? Thanks!
> 
> ```
> pip install --upgrade html2image
> ```

Everything appears to be working again.  Thanks!
